mlir/include/mlir/Dialect/Linalg/IR/LinalgNamedStructuredOps.yaml

```diff
@@ -4096,38 +4096,39 @@ structured_op: !LinalgStructuredOpConfig
     name: I
     kind: input_tensor
     type_var: T1
-    shape_map: affine_map<()[s0, s1, s2, s3, s4, s5, s6, s7, s8, s9] -> (s0, s9, s1 *
-      s2 + s3 * s4, s5 * s6 + s7 * s8)>
+    shape_map: affine_map<()[s0, s1, s2, s3, s4, s5, s6, s7, s8, s9] -> (s0, s1, s2
```

Comment:
@gargaroff - you've seen unrelated changes like these before when running the script to apply `core_named_ops.py`. Do you know why it happens?

Reply:
I haven't investigated this in full, but it looks to me like that this file is simply out-of-date and somebody forgot to update it.

Reply:
Thanks for the reminder. @flemairen6 - please do not bring in these unrelated changes as we don't want to be more different from upstream than is essential.

Reply:
Thanks for the reminder. @flemairen6 - please do not bring in these unrelated changes as we don't want to be more different from upstream than is essential.